src/opensearch.py

```diff
@@ -86,17 +87,28 @@ def is_service_started(self, paused: Optional[bool] = False) -> bool:
 
         # Now, we must dig deeper into the actual status of systemd and the JVM process.
         # First, we want to make sure the process is not stopped, dead or zombie.
-        # From: https://github.com/torvalds/linux/blob/master/fs/proc/array.c#L126-L140
-        # "Parked" state is ignored as it applies to threads.
         try:
-            pid = subprocess.check_output(["lsof", "-ti:9200"], text=True).rstrip()
+            pid = run_cmd("lsof", args="-ti:9200").rstrip()
```

Comment:
rstrip is not necessary - as it is already cleaned in `run_cmd`

Reply:
@Mehdi-Bendriss I will clean up on a next PR